src/install/mod.rs

```diff
@@ -164,7 +164,7 @@ pub fn download_prebuilt(
 fn prebuilt_url(tool: &Tool, version: &str) -> Result<String, failure::Error> {
     let target = if target::LINUX && target::x86_64 {
         match tool {
-            Tool::WasmOpt => "x86-linux",
```

Comment:
This previously matched what `target::x86` fetches

Is 32 bit x86 no longer packaged? ie does the `else if` after this block fail?

Reply:
Yeah, I can only see `x86_64` in the binaryen releases: https://github.com/WebAssembly/binaryen/releases/tag/version_108